WooCommerce/src/main/kotlin/com/woocommerce/android/ui/main/MainPresenter.kt

```diff
@@ -49,6 +56,18 @@ class MainPresenter @Inject constructor(
         mainView = view
         dispatcher.register(this)
         ConnectionChangeReceiver.getEventBus().register(this)
+
+        CoroutineScope(coroutineDispatchers.io).launch {
+            selectedSite.observe()
+                .collect { siteModel ->
+                    if (siteModel != null) {
+                        sendTelemetry.invoke(
+                            appVersion = BuildConfig.VERSION_NAME,
+                            siteModel = siteModel
+                        )
```

Comment:
This is more a question than a change request, but I'm curious if there's a reason why we shouldn't just call the invoke as the operator it's defined to take advantage of the use case naming convention, something like this:
```suggestion
                        sendTelemetry(
                            appVersion = BuildConfig.VERSION_NAME,
                            siteModel = siteModel
                        )
```

Reply:
I agree, thanks for noticing this! I don't know why I called it by `invoke`. Fixed in 1569be2185f589762210cdfa98b5aa06f7c3a0f9

Reply:
Thanks! 🙇